src/player.h

```diff
@@ -7,6 +7,7 @@
 #include "player_activity.h"
 #include "weighted_list.h"
 #include "game_constants.h"
+#include "vitamin.h"
```

Comment:
You don't need to include "vitamin.h" here, you can just forward declare `vitamin` and `vitamin_id` and include the header in player.cpp only.